### ISSUE TITLE
[bug] volume name should be validated before Create

### ISSUE THREAD

**Issue by mathspanda** [alibaba/pouch#2416]
### Ⅰ. Issue Description
As the title desc, volume name should be validated before [Create](https://github.com/alibaba/pouch/blob/master/storage/volume/modules/local/local.go).

### Ⅱ. Describe what happened
```
root@lin-vm1:~# pouch volume create --name "abc def"
Name:         abc def
Scope:
Status:       map[size:]
CreatedAt:    2018-11-2 17:54:38
Driver:       local
Labels:       map[]
Mountpoint:   /var/lib/pouch/volume/abc def
```

### Ⅲ. Describe what you expected to happen
The volume name should be "^[a-zA-Z0-9][a-zA-Z0-9_.-]$".

### IⅤ. Anything else we need to know?
None

### V. Environment:

- pouch version (use `pouch version`):
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by mathspanda** [comment]
If need, i will fix it.

**Comment by zhuangqh** [comment]
> If need, i will fix it.

already fix in #2394

**Comment by mathspanda** [comment]
OK, got it.
